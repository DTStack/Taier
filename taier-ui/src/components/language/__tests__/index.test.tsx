import { render } from '@testing-library/react';
import Language from '..';

jest.mock('@/context', () => {
    const react = jest.requireActual('react');
    return react.createContext({
        supportJobTypes: [{ key: 1, value: 'spark' }],
    });
});

describe('Test Language Component', () => {
    it('Should match snapshot', () => {
        expect(
            render(<Language current={{ tab: { id: 1, data: { id: 1, taskType: 1 } } }} />).asFragment()
        ).toMatchSnapshot();
    });

    it('Should return null', () => {
        expect(render(<Language />).asFragment().textContent).toBe('');
    });
});
