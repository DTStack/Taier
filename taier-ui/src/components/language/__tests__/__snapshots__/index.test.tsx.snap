// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Language Component Should match snapshot 1`] = `
<DocumentFragment>
  <span>
    spark
  </span>
</DocumentFragment>
`;
